notebooks/official/stable_diffusion/dreambooth_stablediffusion.ipynb

```diff
@@ -0,0 +1,1326 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    CUSTOM_PREDICTOR_IMAGE_URI = f"us-docker.pkg.dev/vertex-ai/prediction/pytorch-gpu.1-12:latest"

Why you name this as CUSTOM, when it is a prebuilt image?
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1491/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>

Reply:
Changed.